### ISSUE TITLE
Implement feature flag support for the new SCA payment features

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#5970]
We can deploy the new SCA payment features in parallel with the pre-SCA payment features.

Accessing the new stuff involves using a new `/v2` route in the payment-server front-end.

We would like a way to easily flip between the pre-SCA scenario over to the new post-SCA scenario. This could help us have a roll-back option in case the new approach surfaces issues after being open to the public.

Might also be nice to have a QA-accessible way to flip back and forth in a single testing session - i.e. maybe a cookie or a localStorage flag rather than a site config setting that takes a re-deploy to change.

This is in advance of FXA-2275 and FXA-2229 - presumably we'll have tested & QA'ed & proven out the new routes via feature flipping before finally pulling out the old stuff.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-2301)
┆Issue Number: FXA-2301

**Comment by data-sync-user** [comment]
➤ Chelsea Lewis commented:

+1 to the nice to have on a QA-accessible flip for a testing session, if it's not terribly cumbersome to add. If we build something like that out, this would be important to add to the PI ticket for QA's planning 😀

**Comment by data-sync-user** [comment]
➤ Les Orchard commented:

Worst case even without this issue being resolved, we can tell QA to manually edit the URL for a product or subscription management. Would be better if we could offer a button to push or something to set/clear a cookie.
